### PR TITLE
style(frontend): add subtitle to Oisy logo when logged out

### PR DESCRIPTION
**Pull request by AntonioVentilii-DFINITY**
# Motivation

We want to include the subtitle "NEVER DOWNLOAD AN APP" under the Oisy logo when the user is not logged in.

**It will not be shown in the mobile version.**

The objective is:

Screenshot 2024-07-19 at 10 00 19

# Results

Screenshot 2024-07-19 at 09 56 40

P.S. it is a little too close to the "Alpha" warning, but since this new branch implies the removal of such a warning, I will ignore it.

### CODE REVIEW

File: scripts/i18n.mjs [hunk 1]
```diff
@@ -11,7 +11,8 @@ const PATH_TO_OUTPUT = join(PATH_FROM_ROOT, 'lib', 'types', 'i18n.d.ts');
  * Generates TypeScript interfaces from the English translation file.
  */
 const generateTypes = async () => {
-	const { default: en } = await import(PATH_TO_EN_JSON, { assert: { type: 'json' } });
+	// https://github.com/tc39/proposal-import-attributes
```
Comment (peterpeterparker) [inline scripts/i18n.mjs:14]: ?

File: src/frontend/src/lib/components/hero/Header.svelte [hunk 2]
```diff
@@ -21,6 +22,13 @@
 	{:else}
 		<div class="flex p-4 items-center text-off-white">
 			<OisyWalletLogo />
+			{#if authNotSignedIn}
+				<div
+					class="absolute max-w-[3rem] leading-none text-[8px] uppercase font-semibold translate-x-[112%] translate-y-[130%]"
```
[the remark this answers is not in the document]
Comment (peterpeterparker) [reply src/frontend/src/lib/components/hero/Header.svelte:27]: Artem is fine not displaying the slogan on mobile devices, so I guess we do not need those anymore?

[the remark this answers is not in the document]
Comment (AntonioVentilii-DFINITY) [reply src/frontend/src/lib/components/hero/Header.svelte:27]: i still need to show them on the big ones no? I was translating to make it stay right under the "OISY" word

do you have any suggestion on how to do that?

File: scripts/i18n.mjs (hunk 1 above)
Comment (AntonioVentilii-DFINITY) [reply scripts/i18n.mjs:14]: my mistake... this shouldn't have been here

File: src/frontend/src/lib/components/hero/Header.svelte [hunk 3]
```diff
@@ -21,6 +22,13 @@
 	{:else}
 		<div class="flex p-4 items-center text-off-white">
 			<OisyWalletLogo />
+			{#if authNotSignedIn}
```
Comment (peterpeterparker) [inline src/frontend/src/lib/components/hero/Header.svelte:25]: The subscription `$` is missing

File: src/frontend/src/lib/components/hero/Header.svelte [hunk 4]
```diff
@@ -21,6 +22,13 @@
...
+				<div
+					class="absolute max-w-[3rem] leading-none text-[8px] uppercase font-semibold translate-x-[112%] translate-y-[130%] hidden md:flex"
```
Comment (peterpeterparker) [inline src/frontend/src/lib/components/hero/Header.svelte:27]: - For consistency, can we use a standard font size, like text-sm, and not introduce en discrepancy?

- Shouldn't this section be an `::after` that way it does not push the content given that we display it only on wider screen?

**Review by peterpeterparker — COMMENTED** [review]
Can you also update the screenshot in the PR as the slogan is not displayed on mobile devices anymore?

File: src/frontend/src/lib/components/hero/Header.svelte (hunk 3 above)
Comment (AntonioVentilii-DFINITY) [reply src/frontend/src/lib/components/hero/Header.svelte:25]: thank you!

File: src/frontend/src/lib/components/hero/Header.svelte (hunk 4 above)
Comment (AntonioVentilii-DFINITY) [reply src/frontend/src/lib/components/hero/Header.svelte:27]: - the smallest tailwind size` text-xs` is 12px; I used it at first but I thought it was a bit too much

### 12px
Screenshot 2024-07-22 at 11 12 30

### 8px
Screenshot 2024-07-22 at 11 12 57

WDYT?


- I am not really sure how to do that? you mean to create a new class

```css
.text-never-download::after {
    content: {$i18n.hero.text.never_download}
    position: absolute;
    ....
}
```

and then use it as

```svelte
<div class="flex p-4 items-center text-off-white"
class:text-never-download={$authNotSignedIn}
>
	<OisyWalletLogo />
</div>
```

Comment (peterpeterparker) [reply src/frontend/src/lib/components/hero/Header.svelte:27]: Thanks for trying. Let's keep it as such then.

Comment (peterpeterparker) [reply src/frontend/src/lib/components/hero/Header.svelte:27]: That said, didn't Artem agree to not display this slogan on mobile? 
Not sure what was the last final decision?

```
class="hidden md:block"
```

**Review by peterpeterparker — APPROVED** [review]
LGTM, already approving. Just added a last question.

File: src/frontend/src/lib/components/hero/Header.svelte (hunk 4 above)
Comment (AntonioVentilii-DFINITY) [reply src/frontend/src/lib/components/hero/Header.svelte:27]: indeed in the end it was decided to not showing it. corrected!

**Pull request merged**